mechanicalsoup/form.py

```diff
@@ -277,6 +277,45 @@ def set(self, name, value, force=False):
             return
         raise LinkNotFoundError("No valid element named " + name)
 
+    def __getitem__(self, name):
+        """Forward arguments to :func:`~Form.get` to get value for a given form field.
+        For example, :code:`name = form["name"]` calls :code:`form.get("name")`
+        .
```

Comment:
Any reason this period is on its own line? Should probably just be on the previous line.